src/content/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring.mdx

```diff
@@ -243,13 +241,9 @@ In order to obtain accurate browser performance metrics, our browser monitoring
   If your website does not allow any third-party hosted JavaScript to run, you must request a CSP exception via your organization’s standard procedures in order to install the browser monitoring agent.
 </Callout>
 
-When requesting a CSP exception, follow your organization's standard procedures to contact your web or security team.
-
-The browser JavaScript agent requires the following CSP exceptions in order to run properly. Alternatively, you can add all of these exceptions to the fallback `default-src` directive (instead of `script-src` and `connect-src`).
+When requesting a CSP exception, follow your organization's standard procedures to contact your web or security team. The CSP exceptions for your use case depend the browser agent version you're currently running. We always recommend updating to the latest possible version of the browser agent.
 
-<Callout variant="important">
-In older browsers `image-src` or requests for images can be a possible exception that should be accounted for.
-</Callout>
+You can add all of these exceptions to the fallback `default-src` directive (instead of `script-src` and `connect-src`). The browser JavaScript agent requires the following CSP exceptions: 
```

Comment:
```suggestion
You can add all of these exceptions to the fallback `default-src` directive (instead of `script-src` and `connect-src`). The browser agent requires the following CSP exceptions: 
```